File: 03_c/day02_largest_number.c
#include <stdio.h>

int main() {
    int a, b, c;
    int largest;

    printf("Enter three numbers :\n");
    scanf("%d %d %d", &a, &b, &c);

    largest = a;
    
    if(b > largest) {
        largest = b;
    }
    
    if(c > largest) {
        largest = c;
    }

    printf("Largest number is %d\n", largest);
    
    return 0;
}

File: 05_csharp/Day02_Largest_Number.cs
using System.Runtime.InteropServices;

public class LargestNumber
{
    public static void Run()
    {
        Console.Write("Enter first number :");
        int a = Convert.ToInt32(Console.ReadLine());

        Console.Write("Enter second number :");
        int b = Convert.ToInt32(Console.ReadLine());

        Console.Write("Enter third number :");
        int c = Convert.ToInt32(Console.ReadLine());

        int largest;

        if(a >= b && a >= c)
        {
            largest = a;
        }
        else if(b >= a && b >= c)
        {
            largest = b;
        }
        else
        {
            largest = c;
        }
        Console.WriteLine("Largest number is " + largest);
    }
}